languages/elixir/concepts/ranges/about.md

```diff
@@ -1,16 +1,22 @@
 [Ranges][range] represent a sequence of one or many consecutive integers. They:
 
 - Are created using the [`..` operator][range-operator] and can be both ascending or descending.
-- Are inclusive of the last value.
+- Are inclusive of the first and last values.
 - Implement the [_Enumerable protocol_][enumerable].
 - Are represented internally as a struct, but can be pattern matched using `..`.
-- Can be used with integers written in the binary, octal, hexadecimal, and codepoint notation.
+- Can be used with integers written in the binary, octal, hexadecimal, and code point notation.
+- Can be turned into lists with functions such as [`Enum.to_list/1`][enum-to-list] or [`Enum.map/2`][enum-map].
 
 ```elixir
-Enum.map(?D..?A, & &1 + 3)
-# => 'GFED'
+Enum.to_list(9..1)
+# => [9, 8, 7, 6, 5, 4, 3, 2, 1]
+
+Enum.map(?A..?F, & <<&1>>)
+["A", "B", "C", "D", "E", "F"]
```

Comment:
Is there a reason to use `map` over `to_list`? ie, would I know which to use in the exercise based on stuff I've learnt previously?

Reply:
to_list won't do what you think it will. It only enumates the string to a proper list

Reply:
```
[?A, ?B] == 'AB'    # A charlist
<<?A, ?B>> == "AB"  # A string
```

Reply:
To answer your other  question though, people should know about the `Enum` library of functions, since this exercise depends on completing the `enum` concept exercise first

Reply:
Ah, cool. Thanks :)